nixos/modules/services/mail/rmilter.nix

```diff
@@ -157,13 +176,9 @@ in
           type = types.str;
           description = "Addon to postfix configuration";
           default = ''
-smtpd_milters = ${head allSockets}
-# or for TCP socket
-# # smtpd_milters = inet:localhost:9900
-milter_protocol = 6
-milter_mail_macros = i {mail_addr} {client_addr} {client_name} {auth_authen}
-# skip mail without checks if milter will die
-milter_default_action = accept
```

Comment:
For what it's worth, I don't use this feature of the package.  I add the milter stuff to postfix by hand because my needs are a bit different.